api_docs/deploying-bots.md

```diff
@@ -117,55 +117,44 @@ Botserver process.  You can do this with the following procedure.
 
 {start_tabs}
 
-1. Download the `botserverrc` from the `your-bots` settings page, using
-   the "Download config of all active outgoing webhook bots in Zulip
-   Botserver format." option at the top.
+1. Download the `botserverrc` from your Bot settings page, by following
+   the first 3 steps from [these instructions][zuliprc], and using the
+   "**Download config of all active outgoing webhook bots in Zulip Botserver
+   format**" option.
```

Comment:
This option is only shown if the user has at least one "Outgoing" webhook bot configured, so that might be worth noting somewhere here (e.g. in the dev environment it's only shown for aaron).